File: day1/solve.py
def part1(input):
    return max(sum(int(l) for l in e.split('\n')) for e in input.split('\n\n'))

def part2(input):
    elves = [sum(int(l) for l in e.split('\n')) for e in input.split('\n\n')]
    return sum(sorted(elves)[-3:])

File: day1/solve.rb
def part1(input)
  input.split("\n\n").map { |l| l.split("\n").sum(&:to_i) }.max
end
    

def part2(input)
  part2 = input.split("\n\n")
    .map { |l| l.split("\n").sum(&:to_i) }
    .sort.reverse.slice(0, 3).sum
end